rules.mk.in: Don't set prefix, datadir, libdir, datarootdir and repdir.
rules.mk.in: Use pkg-config to get the absolute path for repcommonexedir.

Merged into upstream: <https://github.com/SawfishWM/librep/commit/a1f2db721aa5055e90f6a76fde625946340ed8cf>.

--- librep_0.92.4.orig/rules.mk.in	2015-01-31 23:21:59.614751496 +0800
+++ librep_0.92.4/rules.mk.in	2015-02-02 18:56:55.097171657 +0800
@@ -1,18 +1,10 @@
 # rules.mk
 
-prefix=@prefix@
-datadir=@datadir@
-libdir=@libdir@
+repcommonexecdir?=$(shell pkg-config --variable=repcommonexecdir librep)
+rpath_repcommonexecdir:=$(repcommonexecdir)
 
-# shut up configure
-datarootdir=@datarootdir@
-
-repdir=@repdir@
-repcommonexecdir=@repcommonexecdir@
-rpath_repcommonexecdir=@repcommonexecdir@
-
-rep_LIBTOOL=$(repcommonexecdir)/libtool --tag CC
-rep_INSTALL_ALIASES=$(repcommonexecdir)/install-aliases
+rep_LIBTOOL:=$(repcommonexecdir)/libtool --tag CC
+rep_INSTALL_ALIASES:=$(repcommonexecdir)/install-aliases
 
 # use this like:
 # foo.la : foo.lo bar.lo
